frontend/src/app/render_wasm/api/fonts.cljs

```diff
@@ -91,17 +91,18 @@
             (aget id-buffer 3)
             (:weight font-data)
             (:style font-data)
-            emoji?)
+            emoji?
+            fallback?)
     true))
 
 (defn- store-font-url
-  [font-data font-url emoji?]
+  [font-data font-url emoji? fallback?]
   (->> (http/send! {:method :get
                     :uri font-url
                     :response-type :blob})
```

Comment:
You can use `:response-type :buffer` and avoid the `wapi/read-file-as-array-buffer`